subjects/flags/README.md

```diff
@@ -9,6 +9,10 @@ This program should :
 - Insert the string given to the `--insert` (or `-i`), into the argument `string`, if given.
 - If the flag `--order` (or `-o`) is given, order the `string` argument (in ASCII order).
 - If there are no arguments or if the flag `--help` (or `-h`) is given, the options should be printed as in the example.
+  - The short flag will have two spaces before the (-).
+  - The explanation of the flag will have a tab followed by a space before the beginning of the sentence (This flag...).
```

Comment:
These rules seem unnecessarily convoluted. Why not simply a space for the short flags and a tab for the explanation, without the spurious extra space?

Reply:
This is how it was before when someone did the exercise, i just added an explanation, but yeah i agree its a bit unnecessary. But it is also nice to give the students some traps so i think its ok to let this one be like it is